pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -78,10 +78,10 @@ type dnsEntry struct {
 
 const (
 	maxBridgeNumber int    = 256
-	timeoutSec      int32  = 150    // less than 150 sec, consider done
-	markMask        uint32 = 0xffff // get the Mark bits for ACL number
-	appShiftBits    uint32 = 24     // top 8 bits for App Number
-	maxFlowPack     int    = 280    // approximate 100 bytes per flow/dns, get this under 30k
+	timeoutSec      int32  = 150      // less than 150 sec, consider done
+	markMask        uint32 = 0xffffff // get the Mark bits for ACL number
```

Comment:
Doesn't this mean the controller can send 0xFFFFFF which will be interpreted as DropMarkValue elsewhere in the code? I think you need to either disallow that value from the controller, or use 0xFFFFFFFF as the dropmark.

Reply:
Controller currently allocates 16bit acl ids (LS 2 bytes). MSB is reserved for app number and the subsequent 8 bits are left for future use.